models/gc_picai_baseline/config/default.yml

```diff
@@ -0,0 +1,35 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Prostate MRI classification default (dicom to json)
+
+execute:
+- FileStructureImporter
+- MhaConverter
+- PicaiBaselineRunner
+- ReportExporter
+- DataOrganizer
+
+modules:
+  FileStructureImporter:
+    input_dir: input_data
+    structures:
```

Comment:
Agree that this cannot be extracted trustfully from DICOM.

Some comments:
- the `images` subfolder is not required and should be removed.
- `mod=mr` must be set for all data, which is modality MR.
- you can introduce a model-specific meta key to distinguish between the three models.
- The sub-folder names contain redundant information that might be removed (`...-prostate-mri`)
- You can utilize the subfodler name (where the dicom is contained) to generate a meta key from that folder, e.g., `$sid/$type@dicom:mod=mr`